### PR TITLE
Fix Emission Output Variable Names "Environmental Impact OtherFuel1 CO2 Water Consumption Volume" and "Environmental Impact OtherFuel2 CO2 Water Consumption Volume"

### LINKED ISSUE #9069: Report variable Environmental Impact OtherFuel1 CO2 Water Consumption Volume makes no sense
Issue overview
--------------
Recent review of emissions reports shows a strange inference to CO2 Water Consumption Volume. The V9.5 and V9.6 both show Environmental Impact OtherFuel1 Water Consumption Volume as the report name. Yet the latest code shows:

PollutionModule.cc 
    SetupOutputVariable(state, "Environmental Impact OtherFuel1 CO2 Water Consumption Volume"
    SetupOutputVariable(state, "Environmental Impact OtherFuel2 CO2 Water Consumption Volume"

image

### Details
Some additional details for this issue (if relevant):
 - Platform (Operating system, version)
 - Version of EnergyPlus (if using an intermediate build, include SHA)
 - Unmethours link or helpdesk ticket number

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Defect file added (list location of defect file here)
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

### PR DESCRIPTION
**Pull request by matthew-larson**
Pull request overview
---------------------
 - Fixes #9069 
 - This pull request corrects the output variable strings "Environmental Impact OtherFuel1 CO2 Water Consumption Volume" and "Environmental Impact OtherFuel2 CO2 Water Consumption Volume" by removing the "CO2" portion of the string.
 - This also creates a new unit test file for PullutionModule.unit.cc and a unit test for the `SetupPollutionMeterReporting` function that checks the output variables.

### Pull Request Author
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [ ] Label the PR with at least one of: Defect, Refactoring, NewFeature, Performance, and/or DoNoPublish
 - [ ] Pull requests that impact EnergyPlus code must also include unit tests to cover enhancement or defect repair
 - [ ] Author should provide a "walkthrough" of relevant code changes using a GitHub code review comment process
 - [ ] If any diffs are expected, author must demonstrate they are justified using plots and descriptions
 - [ ] If changes fix a defect, the fix should be demonstrated in plots and descriptions

### Reviewer
This will not be exhaustively relevant to every PR.
 - [ ] Perform a Code Review on GitHub
 - [ ] If branch is behind develop, merge develop and build locally to check for side effects of the merge
 - [ ] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] CI status: all green or justified
 - [ ] Check that performance is not impacted (CI Linux results include performance check)
 - [ ] Run Unit Test(s) locally

### CODE REVIEW

**Comment by nrel-bot-3** [comment]
@matthew-larson @lgentile it has been 28 days since this pull request was last updated.

File: src/EnergyPlus/PollutionModule.cc [hunk 1]
```diff
@@ -4242,7 +4242,7 @@ void SetupPollutionMeterReporting(EnergyPlusData &state)
                                     _,
                                     "");
                 SetupOutputVariable(state,
-                                    "Environmental Impact OtherFuel2 CO2 Water Consumption Volume",
+                                    "Environmental Impact OtherFuel2 Water Consumption Volume",
```
Comment (Myoldmopar) [inline src/EnergyPlus/PollutionModule.cc:4245]: OK

File: tst/EnergyPlus/unit/CMakeLists.txt [hunk 2]
```diff
@@ -1,273 +1,274 @@
-if(USE_GLYCOL_CACHING)
-  add_definitions("-DEP_cache_GlycolSpecificHeat")
-endif()
+if (USE_GLYCOL_CACHING)
```
Comment (Myoldmopar) [inline tst/EnergyPlus/unit/CMakeLists.txt]: Is there a reason this file was reformatted here?  Should've just been a one line addition for the new unit test file...

**Review by Myoldmopar — CHANGES_REQUESTED** [review]
Could the reformatting be pulled out of here?  If we need to do a broad reformatting of files, let's do it in a single commit on a single PR so we can then add it to the git commit skip file.  I haven't tested this branch, but if I understand the intent, then the tiny fix is quite reasonable.  Let me know if you want me to pull back the changes in that file.

File: tst/EnergyPlus/unit/CMakeLists.txt (hunk 2 above)
Comment (matthew-larson) [reply tst/EnergyPlus/unit/CMakeLists.txt]: CLion kept forcing this on me when I was updating this file for the one line addition.  If it's an issue, I can try to revert this back and do this manually to not effect the formatting.

**Comment by Myoldmopar** [comment]
I'll take out the formatting changes, but this will also need a line in the transition output variables csv file. That file isn't created yet.  I need to make the new version number change, then I'll fix this one up.

**Comment by nrel-bot-3** [comment]
@matthew-larson @lgentile it has been 28 days since this pull request was last updated.

**Comment by Myoldmopar** [comment]
Alright, got the formatting issue taken care of, and added these report variables to the appropriate support file.  I think this one also might be good to go now.

**Review by Myoldmopar — APPROVED** [review]

**Comment by Myoldmopar** [comment]
Pulling develop into this now locally to do final testing but if it passes I'll get this one merged next.

**Comment by Myoldmopar** [comment]
Good to go, pulled develop and tested.  Merging this one too.

**Pull request merged**